[CECO-1] Add parameter to enable/disable operator metrics forwarder (#1192)
DataDog/helm-charts: charts/datadog-operator/CHANGELOG.md

# Changelog

## 1.1.2

> * Add configuration for Operator parameter `operatorMetricsEnabled` : this parameter can be used to disable the Operator metrics forwarder. This component is responsible to send metrics (such as whether or not a feature is enabled) and events related to the Datadog custom resources. It is enabled by default.

Reviewer: ```suggestion
* Add configuration for Operator flag `operatorMetricsEnabled` : this parameter can be used to disable the Operator metrics forwarder. It is enabled by default.
```

suggested a shorter version, in the spirit of keeping the Changelog entries somewhat short
